### PR TITLE
SQL update 2018-11-16 rebased

### PR DESCRIPTION
**Pull request by bjornharrtell**
This is the file from https://github.com/Septima/fikspunktsregister/pull/6 with resolved conflict and `set sqlblanklines on` which might resolve sqlplus issues.

### CODE REVIEW

**Comment by bjornharrtell** [comment]
Looks like `set sqlblanklines on` does resolve the issues loading the schema with sqlplus. I suggest this PR replace #6.

File: test/fixtures/sql/fikspunkt_forvaltning.sql [hunk 1]
```diff
@@ -1,48 +1,49 @@
 CREATE TABLE BEREGNING (
+
    OBJECTID INTEGER GENERATED ALWAYS AS IDENTITY (START WITH 1 INCREMENT BY 1 ORDER NOCACHE) PRIMARY KEY,
    REGISTRERINGFRA TIMESTAMP WITH TIME ZONE NOT NULL,
    REGISTRERINGTIL TIMESTAMP WITH TIME ZONE,
-   SAGSEVENTID VARCHAR2(4000) NOT NULL
+   SAGSEVENTID VARCHAR2(36) NOT NULL
```
Comment (kbevers) [inline test/fixtures/sql/fikspunkt_forvaltning.sql]: I realize that `VARCHAR(4000)` is a bit excessive in most places here, but why change it here? Seems like something that should be fixed upstream.

**Comment by bjornharrtell** [comment]
Great I'll merge, but first trying to find a solution without having to add `set sqlblanklines on` to the schema sql..

**Comment by bjornharrtell** [comment]
Finally got it to run.. but the schema has issues, at least as parsed by sqlplus. 

Partial output:

```
UNIQUE (ID)
  *
ERROR at line 3:
ORA-02261: such unique or primary key already exists in the table


ALTER TABLE OBSERVATION ADD
*
ERROR at line 8:
ORA-01735: invalid ALTER TABLE option


SP2-0734: unknown command beginning "IF :new.RE..." - rest of line ignored.
SP2-0734: unknown command beginning "IF :new.SA..." - rest of line ignored.
SP2-0042: unknown command "end" - rest of line ignored.
ALTER TABLE OBSERVATION ADD
*
ERROR at line 8:
ORA-01735: invalid ALTER TABLE option



Trigger created.


Trigger created.


Warning: Trigger created with compilation errors.


Trigger created.


Trigger created.


Trigger created.


Index created.

  PRIMARY KEY
  *
ERROR at line 2:
ORA-02260: table can have only one primary key


ALTER TABLE OBSERVATIONTYPE ADD
*
ERROR at line 15:
ORA-01735: invalid ALTER TABLE option


VALUES ('Koteforskel fra fikspunkt1 til fikspunkt2 (h2-h1) opmålt trigonometrisk' , 'trigonometrisk_koteforskel', TRUE, FALSE, 'Koteforskel [m]', 'Nivellementslængde [m]', 'Antal opstillinger', 'Afstandsafhængig varians pr. målt koteforskel [m^2/m^2]', 'Afstandsuafhængig varians pr. målt koteforskel [m^2]', 'Total varians koteforskel [m^2]', NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL)
                                                                                                                             *
ERROR at line 2:
ORA-00984: column not allowed here


VALUES ('Horisontal retning med uret fra opstilling til sigtepunkt (reduceret til ellipsoiden)' , 'retning', TRUE, FALSE, 'Retning [m]', 'Varians  retning hidrørende instrument, pr. sats  [rad^2]', 'Samlet centreringsvarians for instrument prisme [m^2]', 'Total varians retning [rad^2]', NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL)
                                                                                                                   *
ERROR at line 2:
ORA-00984: column not allowed here


VALUES ('Horisontal afstand mellem opstilling og sigtepunkt (reduceret til ellipsoiden)' , 'horisontalafstand', TRUE, FALSE, 'Afstand [m]', 'Afstandsafhængig varians afstandsmåler [m^2/m^2]', 'Samlet varians for centrering af instrument og prisme, samt grundfejl på afstandsmåler [m^2]', 'Total varians horisontalafstand [m^2]', NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL)
                                                                                                                      *
ERROR at line 2:
ORA-00984: column not allowed here


VALUES ('Skråafstand mellem opstilling og sigtepunkt' , 'skråafstand', TRUE, FALSE, 'Afstand [m]', 'Afstandsafhængig varians afstandsmåler pr. måling [m^2/m^2]', 'Samlet varians for centrering af instrument og prisme, samt grundfejl på afstandsmåler pr. måling [m^2]', 'Total varians skråafstand [m^2]', NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL)
                                                                                       *
ERROR at line 2:
ORA-00984: column not allowed here


VALUES ('Zenitvinkel mellem opstilling og sigtepunkt' , 'zenitvinkel', TRUE, FALSE, 'Zenitvinkel [rad]', 'Instrumenthøjde [m]', 'Højde sigtepunkt [m]', 'Varians zenitvinkel hidrørende instrument, pr. sats  [rad^2]', 'Samlet varians instrumenthøjde/højde sigtepunkt [m^2]', 'Total varians zenitvinkel [rad^2]', NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL)
                                                                             *
ERROR at line 2:
ORA-00984: column not allowed here


VALUES ('Vektor der beskriver koordinatforskellen fra punkt 1 til punkt 1 (v2-v1)' , 'vektor', TRUE, FALSE, 'dx [m]', 'dy [m]', 'dz [m]', 'Afstandsafhængig varians [m^2/m^2]', 'Samlet varians for centrering af antenner [m^2]', 'Total varians [m^2]', 'Varians dx [m^2]', 'Varians dy [m^2]', 'Varians dz [m^2]', 'Covarians dx, dy [m^2]', 'Covarians dx, dz [m^2]', 'Covarians dy, dz [m^2]', NULL, NULL, NULL)
                                                                                                     *
ERROR at line 2:
ORA-00984: column not allowed here
```

**Comment by bjornharrtell** [comment]
Schema loading fixes done in #14, this PR has now been rebased and is apart from the rebase identical to #6.

**Comment by bjornharrtell** [comment]
I'm going to close this one as it's now a matter of fixing the schema.

**Pull request closed without merging**